src/BundleAnalyzerPlugin.js

```diff
@@ -141,16 +141,17 @@ class BundleAnalyzerPlugin {
   }
 
   getBundleDirFromCompiler() {
-    switch (this.compiler.outputFileSystem.constructor.name) {
-      case 'MemoryFileSystem':
-        return null;
-      // Detect AsyncMFS used by Nuxt 2.5 that replaces webpack's MFS during development
-      // Related: #274
-      case 'AsyncMFS':
-        return null;
-      default:
-        return this.compiler.outputPath;
+    if (this.compiler.outputFileSystem.constructor) {
```

Comment:
We need to understand why it's not available now and how to get FS type in webpack 5. @sokra?

Reply:
@th0r we drop `memory-fs` in favor `memfs` package